docs/getting_started/installation.md

```diff
@@ -82,41 +84,72 @@ Then, simply launch the desired script:
 
-    1. Install NVIDIA's [Isaac Sim platform](https://docs.omniverse.nvidia.com/app_isaacsim/app_isaacsim/install_basic.html) (1)
+    3. Export IsaacSim directory path as an environment variable: (3)
 
-    2. Export IsaacSim directory path as an environment variable: (2)
+        On Linux (bash):
+        ```shell
+        export ISAAC_SIM_PATH=<YOUR_PATH_TO_ISAAC_SIM>
+        ```
 
+        On Windows (cmd):
         ```shell
-        export ISAAC_SIM_PATH = <YOUR_PATH_TO_ISAAC_SIM>
+        set ISAAC_SIM_PATH=<YOUR_PATH_TO_ISAAC_SIM>
         ```
 
-    3. Clone [**`OmniGibson`**](https://github.com/StanfordVL/OmniGibson) and move into the directory:
+    4. Clone [**`OmniGibson`**](https://github.com/StanfordVL/OmniGibson) and move into the directory:
 
         ```shell
         git clone https://github.com/StanfordVL/OmniGibson.git
         cd OmniGibson
         ```
 
-    4. Run the command to setup a virtual conda environment to run **`OmniGibson`**:
+    5. Setup a virtual conda environment to run **`OmniGibson`**:
 
+        On Linux (bash):
         ```shell
-        chmod +x setup_conda_env.sh
-        ./setup_conda_env.sh
+        source setup_conda_env.sh
         ```
 
-    5. This will automatically create an dump you into a conda env called `omnigibson`. If you need to activate this environment later, simply call:
+        On Windows (cmd):
+        ```shell
+        setup_conda_env.bat
+        ```
+
+        ??? info "Note for zsh users"
+            bash is **strongly** recommended on Linux. If you are using zsh, you need to change `${BASH_SOURCE[0]}` and `${BASH_SOURCE}` to `$0` in the first line of `<ISAAC_SIM_PATH>/setup_conda_env.sh` and `<ISAAC_SIM_PATH>/setup_python_env.sh` respectively in order for **`OmniGibson`** to work properly.
+        
+    6. This will automatically create an dump you into a conda env called `omnigibson`. If you need to activate this environment later, simply call:
 
         ```shell
         conda activate omnigibson
         ```
 
-    6. 🎉 Congrats! You installed **`OmniGibson`** successfully.  
+    7. Download **`OmniGibson`** assets and datasets:
+
+        On Linux (bash):
+        ```shell
+        OMNIGIBSON_NO_OMNIVERSE=1 python omnigibson/scripts/setup.py
+        ```
+
+        On Windows (cmd):
+        ```shell
+        set OMNIGIBSON_NO_OMNIVERSE=1&&python omnigibson/scripts/setup.py&&set OMNIGIBSON_NO_OMNIVERSE=
+        ```
+
+    8. 🎉 Congrats! You installed **`OmniGibson`** successfully.  
 
 1. Be sure keep track of where you choose Omniverse to write package files! By default this should be `~/.local/share/ov/pkg`
 
-2. If you installed Isaac Sim to the default location, this is `~/.local/share/ov/pkg/isaac_sim-2022.1.1`
+2. Cache and Nucleus is not required for our project. It is sufficient to install Isaac Sim after installing Omniverse Launcher
```

Comment:
I think you can delete this -- Isaac Sim auto installs Cache IIRC, though if it works without cache / nucleus with no noticeable side effects, maybe it's okay? @ChengshuLi do you have cache / nucleus installed on your machine?

Reply:
deleted